### PR TITLE
Probe/Probeset fixes

### PR DESCRIPTION
**Pull request by juettemann**
Bugfix in probeset endpoint, probeset returns now only information about the set, no details about its probes.
Streamlined probe/probeset transcript mappings 
Updated regulatory.info

### CODE REVIEW

File: t/regulatory.t [hunk 1]
```diff
@@ -81,25 +81,26 @@ eq_or_diff($json, $output, 'GET Information about a specific microarray');
...
- sequence        => 'TCTCCTTTGCTGAGGCCTCCAGCTT'  
+  microarray => 'HC-G110',
+  name => '1000_at',          
+  probes => [                         
+    '137:179;'                        
+    ],                                  
+  size => 16                          
   } ;
-$json = json_GET($microarray_probe_set_probe,'JSON: GET Information about a specific probe');
+$json = json_GET($microarray_probe_set,'JSON: GET Information about a specific probe');
```
Comment (avullo) [inline t/regulatory.t]: Should be "about specific probe set"

Comment (juettemann) [reply t/regulatory.t]: Yes, good catch! Any chance you update this, or shall I do a new pull request?

Comment (avullo) [reply t/regulatory.t]: Once you've created the PR, any new commit you do on your fork will be caught by the PR.

**Comment by juettemann** [comment]
One day I remember that! New one in.

**Comment by avullo** [comment]
Nicely done, thanks :+1:

**Pull request merged**